src/tools/web/clusterd/src/app.js

```diff
@@ -1,3 +1,7 @@
+/* app.js
```

Comment:
Please do not repeat the file name as content within the file.

Please use Elektra's standard header, see `doc/CODING.md`.